src/components/ModuleFields/Configurator/User.vue

```diff
@@ -29,7 +50,14 @@ export default {
         { text: this.$t('field.kind.select.optionType.multiple'), value: 'multiple' },
         { text: this.$t('field.kind.select.optionType.each'), value: 'each' },
       ],
+      roleOptions: [],
     }
   },
+
+  mounted () {
+    this.$SystemAPI.roleList().then(({ set: roles = [] }) => {
+      roles.forEach(role => this.roleOptions.push({ name: role.name, roleID: role.roleID }))
```

Comment:
this.roleOptions = roles is good enough and will work the same without going through all the roles